weblate/accounts/tests/test_notifications.py

```diff
@@ -342,6 +344,30 @@ def test_notify_new_announcement(self) -> None:
             "[Weblate] New announcement at Weblate",
         )
 
+    def test_notify_component_translated(self) -> None:
+        unit = self.get_unit()
+        unit.change_set.create(
```

Comment:
Use the same scope as in the real code:

```suggestion
        unit.translation.component.change_set.create(
```